src/notes/alias.service.ts

```diff
@@ -87,7 +87,7 @@ export class AliasService {
     let oldPrimaryId = '';
     let newPrimaryId = '';
 
-    for (const anAlias of note.aliases) {
+    for (const anAlias of await note.aliases) {
```

Comment:
Maybe it would be wise to await this once and then put it in a variable.

Reply:
I'm pretty sure that this awaits `note.aliases` only once :smile: